src/openforms/config/utils.py

```diff
@@ -14,3 +20,26 @@ def remove_empty_design_tokens(obj: dict) -> dict:
         result[key] = updated_value
 
     return result
+
+
+@dataclass
+class ClamAVStatus:
+    can_connect: bool
+    error: Optional[str] = None
```

Comment:
can't we use empty string instead of `None` and have the type always be a string?